configure.ac

```diff
@@ -1115,6 +1115,24 @@ AC_ARG_ENABLE([cryptonly],
 
 AS_IF([test "x$FIPS_VERSION" = "xrand"],[ENABLED_CRYPTONLY="yes"])
 
+# ECH
```

Comment:
@jpbland1 we would like this to be enabled by default with `--enable-all`. Can you make that tweak in the configure.ac?